PurchasesTests/Purchasing/TrialOrIntroPriceEligibilityCheckerTests.swift

```diff
@@ -1,112 +0,0 @@
-//
-//  Copyright RevenueCat Inc. All Rights Reserved.
-//
-//  Licensed under the MIT License (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//      https://opensource.org/licenses/MIT
-//
-//  TrialOrIntroPriceEligibilityCheckerTests.swift
-//
-//  Created by César de la Vega on 9/1/21.
-
-import Nimble
-import XCTest
-@testable import RevenueCat
-import StoreKit
-
-class TrialOrIntroPriceEligibilityCheckerTests: XCTestCase {
-
-    var receiptFetcher: MockReceiptFetcher!
-    var trialOrIntroPriceEligibilityChecker: TrialOrIntroPriceEligibilityChecker!
-    var mockIntroEligibilityCalculator: MockIntroEligibilityCalculator!
-    var mockBackend: MockBackend!
-    var mockProductsManager: MockProductsManager!
-
-    func setup() {
-        receiptFetcher = MockReceiptFetcher(requestFetcher: MockRequestFetcher())
-        mockIntroEligibilityCalculator = MockIntroEligibilityCalculator()
-        mockBackend = MockBackend()
-        let mockOperationDispatcher = MockOperationDispatcher()
-        let identityManager = MockIdentityManager(mockAppUserID: "app_user")
-        let mockProductsManager = MockProductsManager()
-        trialOrIntroPriceEligibilityChecker = TrialOrIntroPriceEligibilityChecker(receiptFetcher: receiptFetcher,
-                                                                                  introEligibilityCalculator: mockIntroEligibilityCalculator,
-                                                                                  backend: mockBackend,
-                                                                                  identityManager: identityManager,
-                                                                                  operationDispatcher: mockOperationDispatcher,
-                                                                                  productsManager: mockProductsManager)
-    }
-
-    func testSK1CheckTrialOrIntroPriceEligibility() {
-        setup()
-        trialOrIntroPriceEligibilityChecker!.sk1CheckEligibility([]) { (eligibilities) in
-        }
-    }
-
-    func testSK1CheckTrialOrIntroPriceEligibilityFetchesAReceipt() {
-        setup()
-        trialOrIntroPriceEligibilityChecker!.sk1CheckEligibility([]) { (eligibilities) in
-        }
-
-        expect(self.receiptFetcher.receiptDataCalled).to(beTrue())
-    }
-
-    func testSK1EligibilityIsCalculatedFromReceiptData() throws {
-        setup()
-        mockIntroEligibilityCalculator.stubbedCheckTrialOrIntroductoryPriceEligibilityCompletionResult = (["product_id": IntroEligibilityStatus.eligible], nil)
-
-        var completionCalled = false
-        var maybeEligibilities: [String: IntroEligibility]?
-        trialOrIntroPriceEligibilityChecker!.sk1CheckEligibility([]) { (eligibilities) in
```

Comment:
I was very confused about this test passing after some changes. Look, we are not passing any productID here nor in line 82 🤦